### ISSUE TITLE
the code is wrong but passed

### ISSUE THREAD

**Issue by zhouchenzc0824** [FreeCodeCamp/freeCodeCamp#12426]
#### Challenge Name

challenge 78

#### Issue Description

run it and it should not be correct 

#### Browser Information

* Browser Name, Version: chorme
* Operating System: win 10
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js
 
<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
<style>
  h2 {
    font-family: Lobster, Monospace;
  }

  .thick-green-border {
    border-color: green;
    border-width: 10px;
    border-style: solid;
    border-radius: 50%;
  }
</style>

<div class="container-fluid">
  <div class="row">
    <div class="col-xs-8">
      <h2 class="text-primary text-center">CatPhotoApp</h2>
    </div>
    <div class="col-xs-4">
      <a href="#"><img class="img-responsive thick-green-border" src="https://bit.ly/fcc-relaxing-cat" alt="A cute orange cat lying on its back. "></a>
    </div>
  </div>
  <img src="https://bit.ly/fcc-running-cats" class="img-responsive" alt="Three kittens running towards the camera. ">
  <div class="row">
    <div class="col-xs-4">
      <button class="btn btn-block btn-primary"><i class="fa fa-thumbs-up"></i> Like</button>
    </div>
    <div class="col-xs-4">
      <button class="btn btn-block btn-info"><i class="fa fa-info-circle"></i> Info</button>
    </div>
    <div class="col-xs-4">
      <button class="btn btn-block btn-danger"><i class="fa fa-trash"></i> Delete</button>
    </div>
  </div>
  <p>Things cats <span class="text-danger">love:</span></p>
  <ul>
    <li>cat nip</li>
    <li>laser pointers</li>
    <li>lasagna</li>
  </ul>
  <p>Top 3 things cats hate:</p>
  <ol>
    <li>flea treatment</li>
    <li>thunder</li>
    <li>other cats</li>
  </ol>
  <form action="/submit-cat-photo">
    
    <div class="row">
      <div class="col-xs-6">
            <label><input type="radio" name="indoor-outdoor"> Indoor</label>
      </div>  
    </div>
    
    <div class="col-xs-6">
      <label><input type="radio" name="indoor-outdoor"> Outdoor</label>
    </div>
    
    <label><input type="checkbox" name="personality"> Loving</label>
    <label><input type="checkbox" name="personality"> Lazy</label>
    <label><input type="checkbox" name="personality"> Crazy</label>
    <input type="text" placeholder="cat photo URL" required>
    <button type="submit">Submit</button>
  </form>
</div>

 
 
```
#### Screenshot

![image](https://cloud.githubusercontent.com/assets/10664598/21745859/2194603c-d4fa-11e6-9901-14d53dce98ca.png)
![issue_20170107165726](https://cloud.githubusercontent.com/assets/10664598/21745877/6bb3e9c6-d4fa-11e6-977f-c4afdc511648.png)

**Comment by Bouncey** [comment]
@zhouchenzc0824 thank you for reporting this issue

The test `Nest all of your radio buttons inside one div with the class row.` is not performing correctly.

The code for this challenge is [here](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/17b2a20a014ec2892bd1535895a20abaebdf3e57/seed/challenges/01-front-end-development-certification/bootstrap.json#L1434)

Please be sure to follow [CONTRIBUTING.md](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/17b2a20a014ec2892bd1535895a20abaebdf3e57/CONTRIBUTING.md)

Happy Coding!

**Comment by alexr101** [comment]
Hey @Bouncey I'll work on this issue if it's allright with you.

**Comment by Bouncey** [comment]
:+1:

**Comment by alexr101** [comment]
Sorry first time contributor here....so I've been trying since last night, but I really don't know what to do. Here's what happens

I set up the app, and it runs fine, but if I run "node seed" at any time - with or without - changes to the challenges code it will break the app. The site will display "something went wrong", and the console will say that can't read value null. 

What I've tried...
I tried restarting the whole process from the cloning stage 2 times, and both times the app breaks when I use node seed. (AFTER I set up everything else in the project. So I'm not skipping steps or anything)

I tried dropping the database via the mongo console, and then doing "node seed". This didn't work either.

I've tried node seed with and without changes to the local code, and nothing.

What am I doing wrong? I pretty much know how to apply the fix, but I just can't get past this block. And at this point I have no idea what I'm looking for either.
